### ISSUE TITLE
Unable to initialize UI5 CLI

### ISSUE THREAD

**Issue by Faiizii** [SAP/openui5-sample-app#212]
I follow the step by step guide. I run all the mentioned commands. However, when I run `ui5` command; I am facing the following issue. 

> Fatal Error: Unable to initialize UI5 CLI
> Error: Cannot find module './shades-classic'
> Require stack:
> - /--/openui5-sample-app/node_modules/@ui5/cli/node_modules/cli-progress/presets/index.js
> - /--/openui5-sample-app/node_modules/@ui5/cli/node_modules/cli-progress/cli-progress.js
>     at Module._resolveFilename (node:internal/modules/cjs/loader:1048:15)
>     at Module._load (node:internal/modules/cjs/loader:901:27)
>     at Module.require (node:internal/modules/cjs/loader:1115:19)
>     at require (node:internal/modules/helpers:130:18)
>     at Object.<anonymous> (/Users/username/Desktop/projects/github/openui5-sample-app/node_modules/@ui5/cli/node_modules/cli-progress/presets/index.js:2:25)
>     at Module._compile (node:internal/modules/cjs/loader:1241:14)
>     at Module._extensions..js (node:internal/modules/cjs/loader:1295:10)
>     at Module.load (node:internal/modules/cjs/loader:1091:32)
>     at Module._load (node:internal/modules/cjs/loader:938:12)
>     at Module.require (node:internal/modules/cjs/loader:1115:19) {
>   code: 'MODULE_NOT_FOUND',
>   requireStack: [
>     '/--/openui5-sample-app/node_modules/@ui5/cli/node_modules/cli-progress/presets/index.js',
>     '/--/openui5-sample-app/node_modules/@ui5/cli/node_modules/cli-progress/cli-progress.js'
>   ]
> }

when I dig down. I realized, shade classic is required but it is missing. I tried to find the npm command to install the classic shade but found nothing. I found shade install npm command and result is saying it is already installed. 
```
const _legacy = require('./legacy');
const _shades_classic = require('./shades-classic'); // this line is producing the issue.
const _shades_grey = require('./shades-grey');
const _rect = require('./rect');

module.exports = {
    legacy: _legacy,
    shades_classic: _shades_classic,
    shades_grey: _shades_grey,
    rect: _rect
};
```

**Comment by RandomByte** [comment]
Hey, could you please share some details on your environment:
 * Node.js Version: 
 * npm Version: 
 * OS/Platform: 

Also, please try removing the `node_modules` directory and also re-install the global UI5 CLI using `npm i -g @ui5/cli`. Thanks!

**Comment by Faiizii** [comment]
Below is my macOS terminal result. 

```
username@user-iMac ~ % node -v
v20.6.1
username@user-iMac ~ % npm -v
\9.8.1
```

I have already installed the ui5 cli globally. However, when I run the version command I see the following result 

```
username@user-iMac ~ % ui5 --v
INFO: Using local @ui5/cli installation

3.6.0 (from /Users/username/node_modules/@ui5/cli/bin/ui5.cjs)
username@user-iMac ~ % 
```

**Comment by RandomByte** [comment]
Sorry for the late reply. I would still suggest to remove the `node_modules` directory in your project and do another `npm install` there. This seems to be an npm-related issue and nothing with the UI5 CLI or the openui5-sample-app.

**Comment by Faiizii** [comment]
This issue was related to file permission managed by OS. I cloned the repo through the `terminal` of the MacOS following the steps `npm i` and `ui5 server`. Somehow, MacOS denying the read permission for some files. I deleted the `node_modules` from the project directory and  `username/.npm/cacache`.  

In the next step I run the npm i through visual studio code and this error disappeared. only deleting `node_modules` from root directory of the project did not do the trick. 

Thank @RandomByte  for suggesting the deleting trick.
